module2/homework/insensitivePalindrom/CMakeLists.txt

```diff
@@ -38,10 +38,10 @@ endif()
 project(insensitivePalindrom)
 enable_testing()
 
-add_executable(${PROJECT_NAME}-ut test.cpp) # add your cpp file here after test.cpp
+add_executable(${PROJECT_NAME}-ut test.cpp checkPalindrome.cpp) # add your cpp file here after test.cpp
```

Comment:
add also line that will compile main.cpp.
`add_executable(${PROJECT_NAME}  main.cpp checkPalindrome.cpp)`